frontend/src/components/AddFundsForm.js

```diff
@@ -0,0 +1,164 @@
+              rows={5}
+              resize='vertical'
+              customClass='js-transaction-description'
+              placeholder='Optional: amount earned by collective from June t-shirt sales'
+              hasError={validationError.notes}
+              value={attributes.notes}
+              onChange={notes => onChange({notes})}
+              />
+          </div>
+        </div>
+        <div className='line1'>donor's information</div>
+        {attributes.fundsFromHost && 
+          <div className='clearfix input-container mb2'>
+            <span> {userString} <span className='AddFundsForm-switch ml1' onClick={::this.switchState}> Change source</span></span>
+          </div>}
+        
+        {!attributes.fundsFromHost && 
+          <div className='clearfix input-container mb2'>
+            <span className='col col-12 pb1'>Tell us who these funds are from. <span className='AddFundsForm-switch ml1' onClick={::this.switchState}> Keep { defaultUser.name || defaultUser.email } as the donor </span></span>
+            <div className='col col-12 sm-col-12 md-col-6 lg-col-6 pr1'>
+              <label>{i18n.getString('name')}</label>
+              <Input
+                customClass='js-transaction-name'
+                hasError={validationError.name}
+                value={attributes.name}
+                handleChange={name => onChange({name})} />
+            </div>
+            <div className='col col-12 sm-col-12 md-col-6 lg-col-6 pl1'>
+              <label>{i18n.getString('email')}</label>
+              <Input
+                customClass='js-transaction-email'
+                hasError={validationError.email}
+                value={attributes.email}
+                handleChange={email => onChange({email})} />
+            </div>
+          </div>}
+
+        <div className='line1'>funds breakdown (in USD)</div>
+        <div className='clearfix input-container mb2'>
+          <div className='col col-12' >
+            <span className='col col-4'> Funding amount: </span> 
+            <span className='col col-2 AddFundsForm-amount'> {formatCurrency(this.state.amount, collective.currency)}</span>
+          </div>
+          <div className='col col-12'>
+            <span className='col col-4 AddFundsForm-bb'> Host fees ({collective.hostFeePercent}%): </span>
+            <span className='col col-2 AddFundsForm-host-fees'> {formatCurrency(this.state.hostFees, collective.currency)} </span>
+          </div>
+          <div className='col col-12'>
+            <span className='col col-4'> Net funds for collective:</span>
+            <span className='col col-2 AddFundsForm-amount'> {formatCurrency(this.state.amount - this.state.hostFees, collective.currency)} </span>
+          </div>
+        </div>
+
+        <p className='notice'>When the source of funds is someone other than the host, we deduct the host fee. By clicking below, you agree to set aside these funds in your bank account on behalf of the collective. </p>
```

Comment:
I like this notice!
I'd add in parenthesis the actual amount to be uber clear: "to set aside those funds ($95)"